ops/services/basic_vm/main.tf

```diff
@@ -41,7 +41,7 @@ resource "azurerm_virtual_machine" "vm" {
   }
 
   os_profile_linux_config {
-    disable_password_authentication = false
+    disable_password_authentication = true
```

Comment:
❓ Out of curiosity, why are we disabling this? 🤔

Reply:
Disabling the password auth will prevent someone from using a password to log into the box directly, which should minimize the chance of a malicious actor being able to brute force their way in. As things stand, someone would have to be on our virtual network before they could have access to this...but, in case the box accidentally gets opened to the outside world, this would only allow SSH access.

Reply:
I see! Thank you for explaining this!